utxonursery.go

```diff
@@ -275,226 +356,156 @@ func (u *utxoNursery) IncubateOutputs(closeSummary *lnwallet.ForceCloseSummary)
 			closeSummary.SelfOutputSignDesc,
 		)
 
-		incReq.outputs = append(incReq.outputs, &selfOutput)
-	}
-
-	// If there are no outputs to incubate, there is nothing to send to the
-	// request channel.
-	if len(incReq.outputs) != 0 {
-		u.requests <- &incReq
+		// We'll skip any zero value'd outputs as this indicates we
+		// don't have a settled balance within the commitment
+		// transaction.
+		if selfOutput.Amount() > 0 {
```

Comment:
this is mostly a defensive check, since we currently don't have logic for handling more complex errors upon publish transaction. it _shouldn't_ happen, but without it we are relying on the input to be well-formed 100% of the time. admitting super low value outputs would cause a lot of headaches down the road, as we currently don't have any way of rejecting invidual outputs from the nursery store in the event of a failed publication